src/daemon/dlt_daemon_common.c

```diff
@@ -295,13 +295,21 @@ int dlt_daemon_applications_clear(DltDaemon *daemon, int verbose)
     return 0;
 }
 
-DltDaemonApplication* dlt_daemon_application_add(DltDaemon *daemon, char *apid, pid_t pid, char *description, int verbose)
+DltDaemonApplication* dlt_daemon_application_add(DltDaemon *daemon,
+                                                 char *apid,
```

Comment:
Please separate indentation/whitespace changes from code changes.
If you are going to reformat, why not reformat all code at once instead?
(synchronized with maintainer @clipka of course).